tests/test_other.py

```diff
@@ -284,7 +284,6 @@ def test_emcc_2(self, compiler, suffix):
         print('skip because windows')
         continue
       target = args[1] if len(args) == 2 else 'hello_world.o'
-      self.clear()
```

Comment:
the clear guarantees we check that the emcc invocation emitted something, so I think it's worthwhile.

Reply:
OK reverting.